easybuild/easyconfigs/i/igraph/igraph-0.7.1-foss-2018b.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'igraph'
+version = '0.7.1'
+
+homepage = 'http://igraph.org'
+description = """igraph is a collection of network analysis tools with the emphasis on 
+efficiency, portability and ease of use. igraph is open source and free. igraph can be 
+programmed in R, Python and C/C++."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/igraph/igraph/releases/download/%(version)s']
+sources = ['igraph-%(version)s.tar.gz']
+checksums = ['d978030e27369bf698f3816ab70aa9141e9baf81c56cc4f55efbe5489b46b0df']
+
+builddependencies = [('Autotools', '20180311')]
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('libxml2', '2.9.8'),
+]
+
+preconfigopts = "aclocal && autoconf && automake && libtoolize && "
```

Comment:
Wouldn't autoreconf do here?

Reply:
Based on https://linux.die.net/man/1/autoreconf, it probably should...